tests/models/llava/test_modeling_llava.py

```diff
@@ -332,3 +332,38 @@ def test_llava_index_error_bug(self):
 
         # Make sure that `generate` works
         _ = model.generate(**inputs, max_new_tokens=20)
+
+    @slow
+    def test_llava_merge_inputs_error_bug(self):
+        # This is a reproducer of https://github.com/huggingface/transformers/pull/28333 and makes sure it does not happen anymore
+        model_id = "llava-hf/llava-1.5-13b-hf"
+        model = LlavaForConditionalGeneration.from_pretrained(model_id)
```

Comment:
This would never fit on our runner, I suggest to load the 7b model in half-precision
```suggestion
    @require_torch_gpu
    def test_llava_merge_inputs_error_bug(self):
        # This is a reproducer of https://github.com/huggingface/transformers/pull/28333 and makes sure it does not happen anymore
        model_id = "llava-hf/llava-1.5-7b-hf"
        model = LlavaForConditionalGeneration.from_pretrained(model_id, torch_dtype=torch.float16, low_cpu_mem_usage=True).to(torch_device)
```

Reply:
Yep